dev/idm-am-common-user/development/README.md

```diff
@@ -0,0 +1,249 @@
+
+    You may also find it useful to import this CA certificate into your operating system's trust store. Consult your OS documentation for how to do so.
+
+8. You can access the platform by opening this URL:
+
+    ```
+    https://login.sample.forgeops.com/console
+    ```
+
+    You can use amadmin / password to login as the am admin.
+    You can use user.0  / password to login as a basic end-user.
+
+    Review the [Access the running platform](../README.md#accessing-the-running-platform) section of the general project README for more details.
+
+9. You can stop the sample any time by just hitting Ctrl^c to exit skaffold. It will automatically remove the running processes.
+
+    You can then rerun the sample by just starting over from step 3.
+
+    You can also remove the whole minikube environment with this command:
+    ```
+    minikube delete
+    ```
+    After doing this, you will need to start over at step 1 if you want to run it again.
+
+## Using GKE
+
+Install the Google Cloud SDK: https://cloud.google.com/sdk/install
+
+1. Login to the GKE dashboard at https://console.cloud.google.com/kubernetes and choose your project from the "Select a project" dialog. If you can't find your project there, talk to your system administrator about setting one up for you to use.
+
+2. Use `gcloud init` to prepare your kubectl environment. After you login, choose the same project that you selected in step 1.
+
+3. Within your browser window from step 1, choose the cluster you want to work with. From there, click on the "Connect" option and copy the "Command-line access" command provided. It should look something like this:
+
+    ```
+    gcloud container clusters get-credentials eng-shared --zone us-east1-c --project engineering-devops
+    ```
+
+    Run this command in your terminal - this will setup kubectl for your use. Afterward, set your kubectl context to use your own namespace (named from your currently-logged-in name):
+
+    ```
+    kubectl config set-context my-context --cluster=`kubectl config current-context` --user=`kubectl config current-context` --namespace=`whoami| sed -e "s/\./_/"`
+    kubectl config use-context my-context
+    ```
+
+4. Login to gcr.io with Docker:
+
+    ```
+    gcloud auth configure-docker
+    ```
+
+5. Use skaffold to build the images and deploy the helm charts.
+
+    This prepares a version of the default skaffold.yaml which is designed to push into your GCR repository and use cluster certificates:
+    ```
+    sed -e "s/engineering-devops/$(gcloud config list --flatten=core --format='value(project)')/" \
+        -e "s/push: false/push: true/" \
+        -e "s/certmanager.acme: false/certmanager.acme: true/" \
+        -e "s/certmanager.issuer: ca-issuer/certmanager.issuer: letsencrypt-prod/" \
+        -e "s/certmanager.issuerKind: Issuer/certmanager.issuerKind: ClusterIssuer/" \
```

Comment:
I added this line to my version recently:

        -e "s/sk-/`whoami| sed -e "s/\./_/"`-sk-/g" \

It allows the helm releases to avoid name conflicts with different people using skaffold to deploy to the same cluster.